scripts/myvectorbench.py

```
@@ -0,0 +1,641 @@
+
+    qps = rows / elapsed if elapsed > 0 else 0.0
+    print(f"    insert_qps = {qps:.0f}")
+    return qps
+
+
+def bench_knn_search(container: Container, vectors: list, wp: dict) -> dict:
+    """Run knn_queries ORDER-BY-distance queries against build_t; return timing metrics."""
+    n_queries = wp.get('knn_queries', 200)
+    print(f"  [knn_search] {n_queries} queries, dim={wp['dim']}")
+
+    rng = random.Random(99)
```

Ruff (0.15.13)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)